event-schema/event_schema.go

```diff
@@ -341,13 +341,17 @@ func (manager *EventSchemaManagerT) handleEvent(writeKey string, event EventT) {
 		}
 
 		if wasOffloaded {
-			manager.reloadModel(offloadedModel)
-			eventModel, ok = manager.eventModelMap[WriteKey(writeKey)][EventType(eventType)][EventIdentifier(eventIdentifier)]
-			if !ok {
-				pkgLogger.Errorf(`[EventSchemas] Failed to reload event +%v, writeKey: %s, eventType: %s, eventIdentifier: %s`, offloadedModel.UUID, writeKey, eventType, eventIdentifier)
-				return
+			err := manager.reloadModel(offloadedModel)
+			if err != nil {
+				eventModel = manager.createModel(writeKey, eventType, eventIdentifier, totalEventModels, archiveOldestLastSeenModel)
+			} else {
```

Comment:
I copied from some existing logic (see for example `else if` below). As agreed this will be moved in a separate PR since we're changing the logic here it doesn't make sense as having it here as part of the linting process :+1:

Reply:
Removed.